src/main/groovy/info/solidsoft/gradle/pitest/PitestPlugin.groovy

```diff
@@ -31,32 +35,31 @@ import org.gradle.api.tasks.Internal
 import org.gradle.api.tasks.SourceSet
 import org.gradle.util.GradleVersion
 
-import static org.gradle.api.plugins.JavaPlugin.TEST_TASK_NAME
-import static org.gradle.language.base.plugins.LifecycleBasePlugin.VERIFICATION_GROUP
-
 /**
  * The main class for Pitest plugin.
  */
+@CompileDynamic
 class PitestPlugin implements Plugin<Project> {
+
     public final static String DEFAULT_PITEST_VERSION = '1.5.0'
     public final static String PITEST_TASK_GROUP = VERIFICATION_GROUP
-    public final static String PITEST_TASK_NAME = "pitest"
+    public final static String PITEST_TASK_NAME = 'pitest'
     public final static String PITEST_CONFIGURATION_NAME = 'pitest'
 
-    private static final String PITEST_JUNIT5_PLUGIN_NAME = "junit5"
+    private static final String PITEST_JUNIT5_PLUGIN_NAME = 'junit5'
 
     @Internal
-    public static final GradleVersion MINIMAL_SUPPORTED_GRADLE_VERSION = GradleVersion.version("5.6") //public as used also in regression tests
-    public static final String PLUGIN_ID = "info.solidsoft.pitest"
+    public static final GradleVersion MINIMAL_SUPPORTED_GRADLE_VERSION = GradleVersion.version('5.6') //public as used also in regression tests
+    public static final String PLUGIN_ID = 'info.solidsoft.pitest'
 
     private final static List<String> DYNAMIC_LIBRARY_EXTENSIONS = ['so', 'dll', 'dylib']
     private final static List<String> DEFAULT_FILE_EXTENSIONS_TO_FILTER_FROM_CLASSPATH = ['pom'] + DYNAMIC_LIBRARY_EXTENSIONS
 
-    private final static Logger log =  Logging.getLogger(PitestPlugin)
+    private final static Logger LOG =  Logging.getLogger(PitestPlugin)
```

Comment:
`log` here is not a constant such as `DEFAULT_PITEST_VERSION`, but a constant reference to the `Logger` class instance. `log` is perfectly fine.

Reply:
Will suppress the warning for this case

Reply:
I hope that it is possible to just add `log` to the exception list (instead of precising the line number it os defined :) ).